### PR TITLE
DEVDOCS-3989: [revise] OrdersV3, revise create an order response

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-3989]

## What changed?
* Revise Create an Order's Response and remove `reason` (x2)
* Editorial clean up
* Schema clean up for Create an Order

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-3989]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3989?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bc-andreadao — APPROVED** [review]
I'm good with it as long as `merchant_calculated_override`  is an array now instead of an object

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -3147,7 +3140,7 @@ components:
                   name: error.expected.jsstring
                   primary_contact.district: error.expected.jsstring.
     503_ServiceUnavailable:
-      description: 'If this occurs, you should retry the request. If you are unable to successfully make a request, please check the BigCommerce system status [here](https://status.bigcommerce.com/). A service is likely down and the request will need to be made again when it is back up (in several hours usually)'
+      description: 'If this occurs, you should retry the request. If you are unable to successfully make a request, please check the BigCommerce system status [here](https://status.bigcommerce.com/). A service is likely down and the request will need to be made again when it is back up (in several hours usually).'
```
Comment (bc-traciporter) [inline reference/orders.v3.yml:3143]:
```suggestion
      description: 'If this occurs, you should retry the request. If you cannot successfully make a request, please check the BigCommerce system status [here](https://status.bigcommerce.com/). A service is likely down, and you will need to make the request again when it is back up (in several hours, usually).'
```

**Review by bc-traciporter — COMMENTED** [review]
I left some suggestions

File: reference/orders.v3.yml [hunk 2]
```diff
@@ -647,7 +637,7 @@ paths:
                 $ref: '#/components/schemas/MetafieldResponse'
         '409':
           description: |
-            The `Metafield` conflicts with another `Metafield`. This can be the result of duplicate unique key combinations of the app's client id, namespace, key, resource_type, and resource_id.
+            The `Metafield` conflicts with another `Metafield`. This can be the result of duplicate unique key combinations of the app's client ID, namespace, key, resource_type, and resource_id.
```
[the remark this answers is not in the document]
Comment (bc-tgomez) [reply reference/orders.v3.yml:640]: Thank you @bc-traciporter; i've created a second PR that captures these changes. DEVDOCS-4863 is the Jira ticket.

**Pull request merged**